modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,127 @@
+
+[NOTE]
+====
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. You can continue to use the deprecated default configuration, but the `openshift-installer` will prompt you with a warning message that indicates the use of deprecated fields in the configuration file. 
+====
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open-source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
+
+.Procedure
+
+. Use the `govc` command-line tool to create vCenter tag categories. The following example generates tags for zones and for regions: 
++
+[source,terminal]
+----
+$ govc tags.category.create -d "<region_tag_category_name>" <region_tag_category>
+
+$ govc tags.category.create -d "<zone_tag_category_name>" <zone_tag_category>
+----
+
+. Create a region tag for each region:
++
+[source,terminal]
+----
+$ govc tags.create -c <region_tag_category> <region_tag>
+----
+
+. Create a zone tag for each zone:
++
+[source,terminal]
+----
+$ govc tags.create -c <zone_tag_category> <zone_tag>
+----
+
+. Attach region tags to each vCenter datacenter object: 
++
+[source,terminal]
+----
+$ govc tags.attach -c <region_tag_category> <region_tag_1> /<datacenter_1>
+----
+
+. Attach the zone tags to each vCenter datacenter object:
++
+[source,terminal]
+----
+$ govc tags.attach -c <zone_tag_category> <zone_tag_1> /<datacenter_1>/host/vcs-mdcnc-workload-1
+----
+
+. Create the `install-config.yaml` file and configure the file to support operating multiple datacenters in a vSphere center.
++
+[IMPORTANT]
+====
+You must specify at least one failure domain for your {product-title} cluster, so that you can provision datacenter objects for your VMware vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on.
+====
+
+. Change to the directory that contains the installation program and initialize the cluster deployment according to your chosen installation requirements.
+
+.Sample `install-config.yaml` file with multiple datacenters defined in a vSphere center
+
+[source,yaml]
+----
+...
+compute:
+...
+  vsphere:
+      zones:
+        - "<zones_tag_1>"
+        - "<zones_tag_2>"
+...
+controlPlane:
+...
+vsphere:
+      zones:
+        - "<zones_tag_1>"
+        - "<zones_tag_2>"
+...
+  vsphere:
+    vcenters:
+...
+    datacenters:
+      - <datacenter1_name>
+      - <datacenter2_name>
+    failureDomains:
+    - name: <zones_tag_1>
+      region: <region_tag_1>
+      zone: <zone_tag_1>
+      server: vcenters[0].server
```

Comment:
is `server: vcenters[0].server` going to be confusing for a user?

maybe something like <vcenter_fqdn> ?

Reply:
Good point. I updated lines 105 and 117 to state:

> server: <fully_qualified_domain_name>